vms/platformvm/README.md

```diff
@@ -0,0 +1,21 @@
+# Platform VM
+
+`PlatformVM` is the virtual machine implementing the [P-chain](https://support.avax.network/en/articles/4058243-what-is-the-platform-chain-p-chain) in `avalanchego`.
```

Comment:
Could we avoid linking externally? I think it would be better if AvalancheGo was a source of truth rather than referencing the support page.

Generally, I think any term that deserves its own support or medium article deserves to be defined within the internal documentation of AvalancheGo.